README.md

```diff
@@ -174,6 +174,34 @@ For a list of available subcommands and options, run:
 
 (If you are not using `uv`, activate your python environment and run the `fip` tool directly.)
 
+
+## Generating `aind-data-schema` metadata
+
+The repository will maintain tools to generate `aind-data-schema` compliant metadata for experiments run using the IsoForce task:
+
+1. Install the `mappers` optional dependencies:
+
+```powershell
+uv sync --extra mappers
+```
+
+2. Run the metadata generation tool via the command line and provide the necessary arguments
+
+```powershell
+uv run fip data-mapper -h
+```
+
+Alternatively, you can use the mapping classes directly. For instance to run the mapper/extractor for acquisition data:
+
+```python
+from aind_physiology_fip.data_mapper import ProtoAcquisitionMapper
```

Comment:
Should be `from aind_physiology_fip.data_mappers import ProtoAcquisitionMapper` (note missing plural in original, "mapper" => "mappers")